src/services/rolePermissionMapping.js

```diff
@@ -86,4 +86,50 @@ module.exports = class modulesHelper {
 			throw error
 		}
 	}
+
+	/**
+	 * list rolePermission.
+	 * @method
+	 * @name list
+	 * @param {Integer} roleId - role_id
+	 * @returns {JSON} - RolePermission list object.
+	 */
+
+	static async list(roleId) {
```

Comment:
Argument name should be in plural since you are passing an array of roles

Reply:
i have changed as roleIds